Meetings/2021-12-08-meeting-minutes.md

```diff
@@ -0,0 +1,49 @@
+# Maintainers' Meeting 2021-21-08
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Calvin A. Allen - @CalvinAllen
+- Dan Siegel - @dansiegel
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Pascal Berger - @pascalberger
+- Nils Andresen - @nils-a
+
+## Recording
+
+**TBD**
+
+### Virtual Inc Staff Observers
+
+- Nicole Miller
+
+## Slides
+
+## Agenda
+
+- Review Action Items
+- Discuss the process to leave the foundation
+
+## Notes
+
+### Process for Leaving the Foundation
+
+- Reviewed the process to leave the Foundation as posted in the [GitHub discussion](https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-1774246).
+- After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts that have been put in place, the board will work with the projects individually that have sent notices to leave.
```

Comment:
This sentence sounds funny to me. (Could well be my English, though.)

> [...] the board will meet with the lawyers to reverse the contracts that have been put in place [...]

Sounds a bit like "all of the contracts". Maybe something like: 

```suggestion
- After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts of the projects that have sent notices to leave. The board will work with each of the aforementioned projects individually.
```